api/src/main/java/io/jsonwebtoken/lang/ImplementationNotFoundException.java

```diff
@@ -0,0 +1,20 @@
+package io.jsonwebtoken.lang;
+
+import io.jsonwebtoken.JwtException;
+
+/**
+ * Exception indicating that no implementation of an jjwt-api SPI was found on the classpath.
+ */
+public class ImplementationNotFoundException extends JwtException {
+
+    private static final String DEFAULT_NOT_FOUND_MESSAGE = "No implementation of %s found on the classpath. Make sure to include an implementation of jjwt-api.";
```

Comment:
Perhaps:

`"Unable to find an implementation for %s using java.util.ServiceLoader.  Ensure you include a backing implementation .jar in the classpath, for example jjwt-impl.jar, or your own .jar for custom implementations."`